src/cincinnati/client.rs

```diff
@@ -205,7 +202,7 @@ impl ClientBuilder {
     }
 
     /// Set (or reset) the HTTP client to use.
-    pub fn http_client(self, hclient: Option<reqwest::Client>) -> Self {
+    pub fn _http_client(self, hclient: Option<reqwest::Client>) -> Self {
```

Comment:
I did not realize this was actually unused. If so, we can simply drop it.

Reply:
Dropping this unused `http_client`